bundles/org.openhab.binding.nikobus/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,142 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="nikobus"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+    <bridge-type id="pc-link">
+        <label>PC-Link</label>
+        <description>PC-Link via serial connection</description>
+        
+        <config-description>
+            <parameter name="port" type="text" required="true">
+                <label>Port</label>
+                <context>serial-port</context>
+                <description>The serial port used to connect to the Nikobus PC Link.</description>
+            </parameter>
+            <parameter name="refreshInterval" type="integer" max="65535" min="10" required="false">
+                <default>60</default>
+                <label>Refresh interval</label>
+                <description>Refresh interval in seconds.</description>
+            </parameter>
+        </config-description>
+    </bridge-type>
+
+	<thing-type id="push-button">
+        <supported-bridge-type-refs>
+            <bridge-type-ref id="pc-link" />
+        </supported-bridge-type-refs>
+
+		<label>Push button</label>
+		<description>A single push button</description>
+
+		<channels>
+			<channel id="button" typeId="button" />
+		</channels>
+
+		<config-description>
+            <parameter name="impactedModules" type="text">
+                <label>List of impacted modules</label>
+                <description>Comma separated list of impacted modules, i.e. 4C6C-1,4C6C-2</description>
+            </parameter>
+		</config-description>
+
+	</thing-type>
+
+	<channel-type id="button">
+      <item-type>Switch</item-type>
+      <label>Button Event</label>
+      <description>Fires when the button is pressed</description>
+	</channel-type>
+	
+	<thing-type id="switch-module">
+      <supported-bridge-type-refs>
+        <bridge-type-ref id="pc-link" />
+      </supported-bridge-type-refs>
+
+      <label>Switch module</label>
+      <description>Nikobus Switch module</description>
+
+      <channels>
+        <channel id="output-1" typeId="switch-output" />
```

Comment:
To make it a little bit more usable in a user interface you can add a label per channel with the number in it:
```suggestion
	<channel id="output-1" typeId="switch-output">
		<label>Output 1</label>
	</channel>
```